src/Order/Base.php

```diff
@@ -810,7 +810,13 @@ public function maybe_create_multi_barcodes( $post_data ) {
 	 */
 	public function maybe_create_shipping_return_barcode( $post_data, $barcode ) {
 		$item_info = new Shipping\Item_Info( $post_data );
```

Comment:
I'm not sure what is the reason of this, and I think its should not be used like this, by this the code will create Item_Info object 3 times, once here, one within `maybe_create_return_barcode` function  and the latest in `create_label` function, this need to be fixed